Sources/NostrSDK/Events/MetadataEvent.swift

```diff
@@ -8,48 +8,66 @@
 import Foundation
 
 /// An object that describes a user.
+/// > Note: [NIP-01 Specification](https://github.com/nostr-protocol/nips/blob/master/01.md#kinds)
 public struct UserMetadata: Codable {
     
     /// The user's name.
     public let name: String?
     
     /// The user's display name.
-    /// > Warning: This property is not part of the Nostr specifications.
+    /// > Note: [NIP-24 Extra metadata fields and tags](https://github.com/nostr-protocol/nips/blob/master/24.md#kind-0)
     public let displayName: String?
 
     /// The user's description of themself.
     public let about: String?
     
     /// The user's website address.
+    /// > Note: [NIP-24 Extra metadata fields and tags](https://github.com/nostr-protocol/nips/blob/master/24.md#kind-0)
     public let website: URL?
     
     /// The user's Nostr address.
     ///
-    /// > Note: [NIP-05 Specification](https://github.com/nostr-protocol/nips/blob/master/05.md#nip-05).
+    /// > Note: [NIP-05 Specification](https://github.com/nostr-protocol/nips/blob/master/05.md#finding-users-from-their-nip-05-identifier).
     public let nostrAddress: String?
     
     /// A URL to retrieve the user's picture.
     public let pictureURL: URL?
     
     /// A URL to retrieve the user's banner image.
+    /// > Note: [NIP-24 Extra metadata fields and tags](https://github.com/nostr-protocol/nips/blob/master/24.md#kind-0)
     public let bannerPictureURL: URL?
-    
+
+    /// A boolean to clarify that the content is entirely or partially the result of automation, such as with chatbots or newsfeeds.
+    /// > Note: [NIP-24 Extra metadata fields and tags](https://github.com/nostr-protocol/nips/blob/master/24.md#kind-0)
+    public let bot: Bool?
+
+    /// The user's LUD-06 Lightning URL (LNURL).
+    /// > Note: [NIP-57 Lightning Zaps](https://github.com/nostr-protocol/nips/blob/master/57.md#protocol-flow)
+    public let lud06: String?
```

Comment:
Suggestion: These names are so cryptic to me. I know it's what's in the spec, but it's hard to understand. What if we called this `lightningURLString` and the lud16 `lightningAddress`? Yes, it would take a lookup into documentation to understand the field it's actually tied to, but it would be much more understandable for a client dev. Up to you though.